src/WebJobs.Extensions.DurableTask/ContextImplementations/RemoteOrchestratorContext.cs

```diff
@@ -73,6 +74,18 @@ internal bool TryGetOrchestrationErrorDetails(out Exception? failure)
 
         internal void SetResult(IEnumerable<OrchestratorAction> actions, string customStatus)
         {
+            // Azure Table Storage enforces a 32 KB limit if a property value is a UTF-16 encoded string.
+            // We apply a 16 KB limit here to align with our in-process model.
```

Comment:
Do we document this anywhere for customers?

Reply:
yeah we have a warning here at the bottom https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-custom-orchestration-status?tabs=csharp, this PR just applies this limit to oop model.